python/sdist/amici/jax/petab.py

```diff
@@ -994,6 +1134,8 @@ def run_simulation(
         nps: jt.Float[jt.Array, "nt *nnp"],  # noqa: F821, F722
         mask_reinit: jt.Bool[jt.Array, "nx"],  # noqa: F821, F722
         x_reinit: jt.Float[jt.Array, "nx"],  # noqa: F821, F722
+        init_override: jt.Float[jt.Array, "np"], # ?? what do these annotations mean ??
```

Comment:
type hints via https://docs.kidger.site/jaxtyping/, supposed to be checked via https://github.com/beartype/beartype but currently not functioning, see https://github.com/AMICI-dev/AMICI/issues/2849

I think the dimension should be `nx` not `np` here.